backend/src/api/model/realm/mutations.rs

```diff
@@ -204,7 +204,7 @@ impl Realm {
         realm.require_moderator_rights(context)?;
 
         let db = &context.db;
-        if name.plain.is_some() == name.block.is_some() {
+        if !realm.is_main_root() && name.plain.is_some() == name.block.is_some() {
```

Comment:
Mhh so you relaxed this condition in order to allow both to be unset? But we still need to catch the case where both are set, which is also wrong for root realms (it's always wrong).